src/compiler/diagnosticMessages.json

```diff
@@ -6598,7 +6598,14 @@
         "category": "Message",
         "code": 90058
     },
-
+    "Export '{0}' from module '{1}'": {

```

Comment:
```suggestion
    "Export \"{0}\" from module \"{1}\"": {
```

minor, seems to match the other ones above this.

Reply:
yup nice, done

Reply:
hmm, the `Remove import from './b'` uses single quotes, what do you think?